tests/lib/rules/jsx-uses-vars.js

```diff
@@ -115,6 +115,22 @@ ruleTester.run('no-unused-vars', rule, {
     }, {
       code: '\
         /*eslint jsx-uses-vars:1*/\
+        var App;\
+        var Hello;\
+        React.render(<App:Hello/>);',
```

Comment:
The transpiler supports them insofar as it parses the AST, and supports custom transforms on that node type. I wouldn't call the lack of an out-of-box transform for namespaced JSX a bug. What would you transform them to anyway? The value I see in them is in the ability to write custom transforms.